.env

```diff
@@ -1,2 +1,4 @@
 TYPE=development
-API_HOST=https://api.staging.sample.com
\ No newline at end of file
+API_HOST=https://api.staging.sample.com
```

Comment:
what's the purpose of API_HOST in this case, since we have API_URL?

Reply:
none I guess. It's a leftover and I'll delete it.